draft-ietf-lamps-dilithium-certificates.md

```diff
@@ -427,7 +429,10 @@ defined in [FIPS204] section 5.4. This specification uses exclusively
 ExternalMu-ML-DSA for pre-hashed use cases, and thus public
 keys identified by `id-hash-ml-dsa-44-with-sha512`,
 `id-hash-ml-dsa-65-with-sha512`, and `id-hash-ml-dsa-87-with-sha512`
-MUST NOT be used in X.509 and related PKIX protocols.
+MUST NOT be used in X.509 and related PKIX protocols with the 
+exception of the Public Key in end-entity X.509 certifacates. 
```

Comment:
I am not married to it. We could remove it.

The point here is that the PK in the leaf cert may be used for Secure Boot or any other use-case we are not aware of. HashML-DSA could be used there. Certs have many uses we are not aware of.